db/db.go

```diff
@@ -0,0 +1,29 @@
+package db
+
+import (
+	"fmt"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+
+	"github.com/iterableio/api/config"
+)
+
+var DB *sqlx.DB
+
+func ConnectSQL() error {
```

Comment:
It's more to be explicit and have control over the order that things are initiated in.